pkg/scheduler/admission/podnodeconstraints/admission.go

```diff
@@ -0,0 +1,77 @@
+package podnodeconstraints
+
+import (
+	"fmt"
+	"io"
+	"io/ioutil"
+	"reflect"
+
+	configlatest "github.com/openshift/origin/pkg/cmd/server/api/latest"
+	"github.com/openshift/origin/pkg/scheduler/admission/podnodeconstraints/api"
+
+	admission "k8s.io/kubernetes/pkg/admission"
+	kapi "k8s.io/kubernetes/pkg/api"
+	kclient "k8s.io/kubernetes/pkg/client/unversioned"
+)
+
+func init() {
+	admission.RegisterPlugin("PodNodeConstraints", func(c kclient.Interface, config io.Reader) (admission.Interface, error) {
+		pluginConfig, err := readConfig(config)
```

Comment:
Hmm, we'd need to make an initial pass through the list of plugins to decide which ones to leave in. And even then, not sure how we decide whether to leave one in or not. Most plugins don't need config.